src/media.js

```diff
@@ -925,7 +902,7 @@
 
               workletAudioSend = new AudioWorkletNode(cthis.audio.Html5Audio.audioContext, 'worklet-audio-send');
 
-              workletAudioSend.onprocessorerror = function (e) {
+              workletAudioSend.onprocessorerror = (e) => {
```

Comment:
Codacy found an issue: ['e' is defined but never used.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=6361181)